LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -732,79 +737,56 @@ - (void)updateViewConstraints {
-                                     attribute: NSLayoutAttributeLeft
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _animatingView
-                                     attribute: NSLayoutAttributeCenterX
-                                    multiplier: 1.0f
-                                      constant: _horizontalGap - 2.0f];
-        NSLayoutConstraint *firstDigitY =
-        [NSLayoutConstraint constraintWithItem: _firstDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        NSLayoutConstraint *secondDigitY =
-        [NSLayoutConstraint constraintWithItem: _secondDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        NSLayoutConstraint *thirdDigitY =
-        [NSLayoutConstraint constraintWithItem: _thirdDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        NSLayoutConstraint *fourthDigitY =
-        [NSLayoutConstraint constraintWithItem: _fourthDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        [self.view addConstraint:firstDigitX];
-        [self.view addConstraint:secondDigitX];
-        [self.view addConstraint:thirdDigitX];
-        [self.view addConstraint:fourthDigitX];
-        [self.view addConstraint:firstDigitY];
-        [self.view addConstraint:secondDigitY];
-        [self.view addConstraint:thirdDigitY];
-        [self.view addConstraint:fourthDigitY];
+        
+        float delta = 10.0f;
+        
+        for (int i = 0; i < _digitsCount; ++i) {
```

Comment:
While not necessarily wrong, the widespread syntax to increment in a for loop is `i++`, could we change them all to that, please? While it doesn't matter in this case, the meaning of  _increment, **then** use `i`_ for `++i` doesn't really fit here.